### ISSUE TITLE
Image Data - Convolutional Neural Net

### ISSUE THREAD

**Issue by karkir0003** [DSGT-DLP/Deep-Learning-Playground#20]
After Iteration 1, we are able to support basic neural networks. However, that might not be interesting. For example, if we have Image Data, CNN (convolutional neural networks) generally are a good start. We should have our drag and drop playground support CNNs and we should allow users to upload image data (and have them mention that the dataset is image data) so that we build image loader and kick off the training. Shouldn't be too much elbow grease to integrate into the backend. 

Layers/Activations/transforms to add: Dropout, Conv2d, BatchNorm2d
